examples/cordova-sample/MyApp/www/index.html

```diff
@@ -37,12 +37,34 @@
     </head>
     <body>
         <div class="app">
-            <h1>Apache Cordova</h1>
-            <div id="deviceready" class="blink">
+            <div id="deviceready" class="blink" hidden="true">
```

Comment:
I'm wondering, is this `div` needed? We are setting `hidden` to `true` here but never setting it to false I think.

Reply:
I removed it and I think the device ready event wasn't being triggered because the configure call wasn't being triggered. I brought it back 🤷

Reply:
haha I went through the same process 😅